.env

```diff
@@ -0,0 +1,39 @@
+# General environment variables
+TZ=London/Europe
+
+# PostgreSQL
+POSTGRES_HOST=postgres
+POSTGRES_PORT=5432
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=knightcrawler
+
+# MongoDB
+MONGODB_HOST=mongodb
+MONGODB_PORT=27017
+MONGODB_DB=knightcrawler
+MONGO_INITDB_ROOT_USERNAME=mongo
+MONGO_INITDB_ROOT_PASSWORD=mongo
+
+# Addon
+DEBUG_MODE=false
+
+# Consumer
+RABBIT_URI=amqp://guest:guest@rabbitmq:5672/?heartbeat=30
+QUEUE_NAME=ingested
+JOB_CONCURRENCY=5
+JOBS_ENABLED=true
+MAX_SINGLE_TORRENT_CONNECTIONS=10
+TORRENT_TIMEOUT=30000
+UDP_TRACKERS_ENABLED=true
+
+# Producer
+RabbitMqConfiguration__Host=rabbitmq
+RabbitMqConfiguration__QueueName=ingested
+RabbitMqConfiguration__Username=guest
+RabbitMqConfiguration__Password=guest
+RabbitMqConfiguration__Durable=true
+RabbitMqConfiguration__MaxQueueSize=0
+RabbitMqConfiguration__MaxPublishBatchSize=500
+RabbitMqConfiguration__PublishIntervalInSeconds=10
+GithubSettings__PAT=
```

Comment:
It's considered a best practice to create a `.env.example` file containing these example values and a `.env` file containing the users' values. You should then add `.env` to `.gitinore` so developers don't accidentally push their tokens to GitHub. 

In summary I suggest to:
- Rename this file to `.env.example`
- Add `.env` to `.gitignore`
- Add an instruction to `README.md` stating users/developers should create a copy of their `.env` file: `cp .env.example .env`.

I can create a PR if you want.

Reply:
done 😄